<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblEstado">
        <property name="text">
         <string>Estado:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboxEstado"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblInicioSim">
        <property name="text">
         <string>Início da Simulação:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dteIniSim">
        <property name="displayFormat">
         <string>dd/MM</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblDataPlantio">
        <property name="text">
         <string>Data do Plantio:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dtePlantio">
        <property name="displayFormat">
         <string>dd/MM</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblAnosDados">
        <property name="text">
         <string>Anos dos Dados Históricos:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="btnAnos">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblSolo">
        <property name="text">
         <string>Tipo de Solo:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cboxSolo"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblCultura">
        <property name="text">
         <string>Cultura:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cboxCultura"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblConfigRegional">
        <property name="text">
         <string>Configuração Regional:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cboxConfReg"/>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="cboxGrupo"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblGrupo">
        <property name="text">
         <string>Grupo:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gbParametros">
      <property name="title">
       <string>Parâmetros</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lblEstoqueInicial">
           <property name="text">
            <string>Estoque Inicial:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="sbEstoqueInicial"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblChuvaLimite">
           <property name="text">
            <string>Chuva Limite:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="sbChuvaLimite"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblMulch">
           <property name="text">
            <string>Mulch:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblRusurf">
           <property name="text">
            <string>RUSURF:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="sbRUSURF"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblReservaUtil">
           <property name="text">
            <string>Reserva Útil:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="sbReservaUtil"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblEscoamentoSup">
           <property name="text">
            <string>Escoamento Superficial:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="sbEscoamentoSup"/>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="sbMulch"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSimular">
      <property name="text">
       <string>Simular</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dwAnos">
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListWidget" name="lvAnos"/>
     </item>
     <item>
      <widget class="QSpinBox" name="sbAnos">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddAno">
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRmAno">
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFechadock">
       <property name="text">
        <string>Fechar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
